lib/thrift/binary/framed/protocol_handler.ex

```diff
@@ -50,6 +64,23 @@ defmodule Thrift.Binary.Framed.ProtocolHandler do
     end
   end
 
+  defp maybe_ssl_accept(socket, 0x16, _optional, ssl_opts, timeout) do
```

Comment:
Would it be useful to define `@ssl_magic_byte 0x16` or similar to self-document this value?